### DISCUSSION TITLE
Mining pool not working with ASICBOOST

### DISCUSSION THREAD

**Discussion by wombatlabs** [coinfoundry/miningcore#1597] [General]
Miningcore is not working with Nicehash's sha256AsicBoost for Bitcoin or Bitcoin Cash. It connects to Nicehash just fine with regular sha256 though.

**Comment by pool2mine** [comment]
Also noticed this... I never digged too much into BTC pools, but recently i was trying to connect an old S9 to MC, i noticed that when using Braiins firmware it connects and get disconnected after a few seconds... (endless loop), then i rollback to bitmain firmware and works fine. I suppose BTC has not been updated recently, but actually... do you think there is any chance for a new pool to run a BTC with the current difficulty?

**Comment by totom9000** [comment]
its not only  btc who use  asic boost  its multiple upand comming coins like mvc (space) who use sha256  and wuld benefit from asic boost.

**Comment by wombatlabs** [comment]


**Reply by Rickyose** [reply]
> I couldn't get AsicBoost to work with this pool. I had to use different software.

hi care to share what software do you use, to make asicboost working ??

**Comment by M1neK1ng** [comment]
it happens with all coins when connected with asicboost.
Ive tried tweaking a few things with no lucky.
hopefully there will be a fix soon to stop random disconnects.

**Comment by mamafunny** [comment]
any luck guys?

**Comment by totom9000** [comment]
You can try  how the asic boost work on mining4people.com as our sha256 do
suport it after code change

**Comment by pool2mine** [comment]
@totom9000 I think the solution is not to send people to mine on your pool, this is an open source community, if you have a solution (and you would like to share) then you're welcome.

@mamafunny seems like Oliver has not take a look yet on this issue, I'm sure he will at some point, in the meantime, i found a working NOMP with asicboost support, i know it's not miningcore, and NOMP API is very old and limited but i have it working for a private pool anyway so it works for me. if it's your case, here is the link for the repo:
https://github.com/meterio/nomp

just adapt it for your coin needs

**Comment by blackmennewstyle** [comment]
Sadly `Oliver` won't be able to have a look on the matter since he officially announced his retirement from future `miningcore` development: https://github.com/oliverw/miningcore/discussions/1687
